parser/parser.go

```diff
@@ -2065,10 +2083,20 @@ func (p *parser) parseRangeExpr(allowCmd bool) ast.Expr {
 
 type tupleExpr struct {
 	ast.Expr
-	items []*ast.Ident
+	Opening token.Pos
+	Items   []ast.Expr
+	Closing token.Pos
 }
 
-func (p *parser) parseLambdaExpr(allowCmd, allowRangeExpr bool) ast.Expr {
+func (p *tupleExpr) Pos() token.Pos {
```

Comment:
Don't implement tupleExpr.Pos/End so that we can found immediately if tupleExpr is leaked into parsing result DOM.

Reply:
remove